be/src/exec/hdfs_scanner.h

```diff
@@ -213,16 +213,16 @@ struct HdfsScannerParams {
 
 struct HdfsScannerContext {
     struct ColumnInfo {
-        int col_idx;
-        TypeDescriptor col_type;
-        SlotId slot_id;
-        std::string col_name;
+        int idx_in_chunk;
```

Comment:
It seems `idx_in_chunk` has not been used, can we delete it?

Reply:
fixed in next pr.